client/css/style.css

```diff
@@ -1129,15 +1129,21 @@ kbd {
 }
 
 #chat .toggle-content .thumb {
-	float: left;
+	display: flex;
```

Comment:
I meant, this would only apply to its children, which .thumb doesn't have any, no?